README.md

```diff
@@ -507,34 +567,82 @@ val toolbarCustomization = ToolbarCustomizationBuilder()
             .headerText("Secure Checkout")
             .buttonText("Close")
             .build()
-            
+
             val uiCustomization = UiCustomizationBuilder()
             .setDefaultTheme(ThemeConfig(
                 toolbarCustomization = toolbarCustomization,
             ))
             .build()
 ```
 
-After the end-user completes the authorizing payment process, the activity result
-callback will be called. Handle it in this manner:
+You can check out the [UiCustomization](/sdk/src/main/java/co/omise/android/config/UiCustomization.kt) class to see customizable UI elements in the challenge flow.
+
+After the end-user completes the payment authorization process, the activity result
+callback will be and you will receive different responses based on how the transaction was processed
```

Comment:
I think we missed 'callback will be **called**' from the original sentence here. Also I think need to add **that** into the 'based on how the transaction **that** was processed'